src/calendar-app/calendar-applicationPaths.ts

```diff
@@ -0,0 +1,54 @@
+// @bundleInto:boot
+
+import { RouteResolver } from "mithril"
+
+export type ApplicationPaths = Record<string, RouteResolver>
+type ViewResolvers = {
+	login: RouteResolver
```

Comment:
some of these you don't need

Reply:
can you elaborate on which ones? We are able to create and pay for accounts with this app so as far as I can tell we need all these.

Reply:
termination, webauthn. not sure about giftcard